README.md

```diff
@@ -1,7 +1,28 @@
-# FortiGate Autoscale Core module
-This is the Autoscale Core module written in TypeScript.
+# Autoscale Core
+The Core Library for the Fortinet Autoscale project.
 
-Note: This is a pre-release for internal development purposes.
+The Autoscale project provides a cloud-based multi-group auto scaling functionality for virtual machines of Fortinet products which have native HA features to form a cluster with failover protection.
+
+This project provides the fundamental, generic logics of a Hybrid Licensing architecture. It also includes implementations for certain popular cloud platforms.
+
+This project has the following features:
+ * Multi-group Hybrid Licensing models. (Hybrid, BYOL-Only, PAYG-Only)
+ * AWS Transit Gateway Integration (see: https://github.com/fortinet/fortigate-autoscale-aws)
```

Comment:
> I feel that this should not be listed here because it is not a part of the CORE project.

The reason it is mentioned here: the CORE provides such functionality for AWS-TGW, regardless whether it is used in a dependent project or not.

Reply:
ok. Thanks for explaining.